include/swift/Driver/Job.h

```diff
@@ -230,9 +230,20 @@ class CommandOutput {
 class Job {
 public:
   enum class Condition {
+    // There was no information about the previous build (i.e., an input map),
+    // or the map marked this Job as dirty
+    // but the input didn't change.
+    // Be maximally conservative with dependencies.
     Always,
+    // The input changed, or this job was non-cascading or [it was not dirty but
+    // primary output was missing].
+    // In other words, this job was scheduled as non-cascading in the last build
+    // but didn't get to run.
```

Comment:
This is only "in other words" to the second clause, so it still seems confusing.

Reply:
Good catch! Have reworded.